app/src/components/RequestTable.js

```diff
@@ -22,65 +26,172 @@ function RequestTable({ requests, token, onSubmit, onWithdraw }) {
+        )}
+      </ContextMenu>,
+    ]
+    return !ownRequests
+      ? [
+          timeColumn,
+          <div
+            css={`
+              display: flex;
+              align-items: flex-start;
+            `}
+          >
+            <LocalIdentityBadge
+              connectedAccount={addressesEqual(requesterAddress, connectedAccount)}
+              entity={requesterAddress}
+            />
+          </div>,
+          ...commonColumns,
+        ]
+      : [timeColumn, ...commonColumns]
+  }
   return (
     <>
       {requests && requests.length > 0 && (
         <DataView
-          fields={['Request Date', 'Deposited', 'Requested', 'Status', 'Actions']}
-          entries={requests
-            .sort(({ date: dateLeft }, { date: dateRight }) =>
-              // Sort by date descending
-              compareDesc(dateLeft, dateRight)
-            )
-            .map(r => [
-              r.requestId,
-              r.date,
-              r.depositAmount,
-              r.depositSymbol,
-              r.depositToken,
-              r.depositName,
-              r.depositDecimals,
-              r.requestAmount,
-              r.status,
-              token.symbol,
-              token.decimals,
-            ])}
-          renderEntry={([
-            requestId,
-            date,
-            depositAmount,
-            depositSymbol,
-            depositTokenAddress,
-            depositName,
-            depositDecimals,
-            requestedAmount,
-            status,
-            requestedSymbol,
-            requestedDecimals,
-          ]) => [
-            <time>{format(date, 'dd/MM/yy')}</time>,
-            <Text>{`${formatTokenAmountSymbol(depositSymbol, depositAmount, false, depositDecimals)} `}</Text>,
-            <Text>{`${formatTokenAmountSymbol(requestedSymbol, requestedAmount, false, requestedDecimals)} `}</Text>,
-            <Status status={status}>{`${status}`}</Status>,
-            <ContextMenu disabled={status != requestStatus.PENDING}>
-              {status === requestStatus.PENDING && (
-                <ContextMenuItem onClick={() => handleSubmit(requestId)}>
-                  <IconWrapper>
-                    <IconVote />
-                  </IconWrapper>
-                  <div css="margin-left: 15px">Submit</div>
-                </ContextMenuItem>
-              )}
-              {status === requestStatus.PENDING && (
-                <ContextMenuItem onClick={() => handleWithdraw(requestId)}>
-                  <IconWrapper>
-                    <IconCoin />
-                  </IconWrapper>
-                  <div css="margin-left: 15px">Withdraw</div>
-                </ContextMenuItem>
-              )}
-            </ContextMenu>,
-          ]}
+          fields={
+            ownRequests
+              ? ['Request Date', 'Deposited', 'Requested', 'Status', 'Actions']
+              : ['Request Date', 'Requester', 'Deposited', 'Requested', 'Status', 'Actions']
```

Comment:
Consider declaring  before render a 
```
const fields = ['Request Date', 'Deposited', 'Requested', 'Status', 'Actions'] 
```

Then do 
```
ownRequests && fields.splice(1,0,'Requester')
```
and pass fields directly as prop